### PR TITLE
f/562-treemap-total-size

### LINKED ISSUE #562: show total db size in tree map
**Is your feature request related to a problem? Please describe:**  
Currently we see the total size of the treemap only in the line chart above.

**Please describe the solution you'd like:**  
We want to have this in the title additionally.

**Affected Component(s):**  
`Frontend`

### PR DESCRIPTION
**Pull request by schTi**
# Resolves #562

**Does your pull request add a feature? Please describe:**  
Adds total size to header label of the treemap.

**Affected Component(s):**  
`Frontend`

### CODE REVIEW

**Comment by MaxSchneider1337** [comment]
I thought about showing the total size like the current metric-value in the other tiles, by using the metric details.

**Comment by schTi** [comment]
Okay, I'm cool with that, gonna do this

**Comment by schTi** [comment]
image

File: hyrisecockpit/frontend/src/services/transformationService.ts [hunk 1]
```diff
@@ -128,7 +128,7 @@ function getStorageData(data: any, primaryKey: string = ""): StorageData {
       size: `${totalDatabaseMemory} MB`,
       encoding: "",
       dataType: "",
-      percentOfDatabase: "100% of total footprint",
+      percentOfDatabase: "",
```
Comment (MaxSchneider1337) [inline hyrisecockpit/frontend/src/services/transformationService.ts:131]: love that change

**Review by MaxSchneider1337 — APPROVED** [review]
hot af

**Pull request merged**